### ISSUE TITLE
False negative on curl command

### ISSUE THREAD

**Issue by sarmong** [pacstall/pacstall#542]
**Describe the bug**

`Curl` command on this line returns 404, whereas the link is valid in browser and using `wget`
 https://github.com/pacstall/pacstall/blob/e5773822209fa3cbc0851674bcc5d7c4f588e992/misc/scripts/install-local.sh#L83

```
ERROR: URL doesn't exist
[!] ERROR: There was an error checking the script!
```

**To Reproduce**

1. Go to https://go.skype.com/skypeforlinux-64.deb - works fine
2. Do `wget https://go.skype.com/skypeforlinux-64.deb` - works fine
3. Do `curl --head --fail https://go.skype.com/skypeforlinux-64.deb`
4. Behold the error

**Expected behavior**
`pacstall` doesn't give false negative results

**Desktop (please complete the following information):**

- OS + version: Ubuntu 20.04
- Pacstall Version: 1.7.2 Carmine
- curl version: 7.68.0

**Comment by D-Brox** [comment]
Can you test it in the current develop branch? We have fixed a issue related to this, I'd like to confirm it fixed this too.
You can change to the develop branch with `pacstall -U pacstall develop`

**Comment by sarmong** [comment]
@D-Brox Yes, installs fine, thanks.

**Comment by D-Brox** [comment]
Closing, as it's going to be released in 1.7.3
